app/lib/theme/widgets/SelectWidget.tsx

```diff
@@ -0,0 +1,48 @@
+import { WidgetProps } from "@rjsf/core";
+import Dropdown from "@button-inc/bcgov-theme/Dropdown";
+import getRequiredLabel from "../utils/getRequiredLabel";
+
+interface Option {
+  type: string;
+  title: string;
+  enum: number[];
+  value: number;
+}
+
+const SelectWidget: React.FunctionComponent<WidgetProps> = (props) => {
+  const { id, onChange, schema, placeholder, label, required, uiSchema } =
+    props;
+
+  const options = schema.anyOf as Array<Option>;
+  return (
+    <>
+      <label htmlFor={`funding-stream-dropdown-${props.id}`}>
+        {getRequiredLabel(label, required)}
+      </label>
+      <Dropdown
+        id={id}
+        onChange={(e) => onChange(e.target.value || undefined)}
+        placeholder={placeholder}
+        size={(uiSchema && uiSchema["bcgov:size"]) || "large"}
+        required={required}
+      >
+        {options.map((opt) => {
```

Comment:
Possibly a bug with the button-inc `Dropdown` component, but it does not include an empty option. It defaults to showing the first option in the options array in the dropdown. This means it will by _look_ like the first option (in this case EP), is selected by default where it is not. The placeholder will never be shown & "EP: will be displayed, but no description will show because it is not actually selected. I suggest adding an empty option above the mapping that shows the placeholder here to avoid confusion,
```suggestion
        <option key={-1} value={undefined}>{placeholder}</option>
        {options.map((opt) => {
```

Reply:
added a placeholder above